dashboard/app/controllers/application_controller.rb

```diff
@@ -137,7 +146,10 @@ def prevent_caching
     :parent_email_preference_opt_in,
     :parent_email_preference_email,
     school_info_attributes: SCHOOL_INFO_ATTRIBUTES,
-  ].freeze
+  ]
+
+  PERMITTED_USER_FIELDS.push(*UI_TEST_ATTRIBUTES) if %i(test development).include? rack_env
```

Comment:
I'd recommend using [`concat`](https://apidock.com/ruby/Array/concat) rather than push+splat, and would also note that we have a helper method [`rack_env?`](https://github.com/code-dot-org/code-dot-org/blob/52a6243c52bdb249dda6777efcc35debb81b26a6/deployment.rb#L23-L26) which can help out here:
```suggestion
  PERMITTED_USER_FIELDS.concat(UI_TEST_ATTRIBUTES) if rack_env?(:test, :development)
```